app/assets/javascripts/discourse/app/controllers/preferences/sidebar.js

```diff
@@ -1,13 +1,26 @@
 import Controller from "@ember/controller";
 import { action } from "@ember/object";
 import { tracked } from "@glimmer/tracking";
+import I18n from "I18n";
 
 import { popupAjaxError } from "discourse/lib/ajax-error";
 
 export default class extends Controller {
   @tracked saved = false;
   @tracked selectedSidebarCategories = [];
   @tracked selectedSidebarTagNames = [];
+  @tracked newSidebarTopicDestination =
+    this.currentUser.user_option.sidebar_topic_destination;
+  @tracked sidebarTopicDestionations = [
```

Comment:
Is there a reason we need this property to be `@tracked` when its value doesn't change?